Dockerfile

```diff
@@ -1,9 +1,14 @@
 FROM quay.io/pypa/manylinux2014_x86_64
 
-RUN yum install npm git python-devel python2-pip gdb -y -q && \
+ARG BAZEL_VERSION
+
+ENV USE_BAZEL_VERSION=$BAZEL_VERSION
+
+RUN yum install npm git python-devel python3-pip gdb -y -q && \
+    npm install -g npm@9.2.0 \
     npm install -g @bazel/bazelisk && \
-    pip install wheel && \
-    pip install --upgrade "pip < 21.0"
+    python3 -m pip install --upgrade --user && \
```

Comment:
Was the pip upgrade beyond 21 necessary? Its a little sketch to use post 21 without full requirements lockdown

Reply:
It was one of the things I was trying when trying to fix the dependencies, but let me see if it works without it!

Reply:
Pip <v21 works! Ive added back that constraint